### DISCUSSION TITLE
Trigger Automation drop down not appearing in Action for Button

### DISCUSSION THREAD

**Discussion by dkmbuild** [Budibase/budibase#12663] [Help] [answered]
Does anyone have a workaround for pushing the Automation dropdown, within action of a button, to appear?  I tried incognito, unpublish/publish, and create a new app for the Automation to appear as a selection.

![image](https://github.com/Budibase/budibase/assets/149087656/0b8680b5-2eac-49e8-9e36-0df42966b275)
![image](https://github.com/Budibase/budibase/assets/149087656/3b23fdef-d63e-4ae7-a892-6f597f57f67a)

**Comment by ConorWebb96** [comment, marked as answer]
Hey @dkmbuild,

What type of automations are these? The only trigger that works with the trigger automation action is the `App action` automation trigger which can be selected whenever creating the automation. I've attached a screenshot below showing the one which needs to be selected in order to populate the options provided in your initial screenshot.

<img width="488" alt="Screenshot 2024-01-03 at 09 04 40" src="https://github.com/Budibase/budibase/assets/126772285/3b0ace2e-c873-4cac-82e3-52dd867b582b">

I hope this makes sense.

**Reply by dkmbuild** [reply]
Yes, this makes sense.  Thank you for your response.
